rllib/utils/replay_buffers/episode_replay_buffer.py

```diff
@@ -518,78 +525,82 @@ def _sample_episodes(
             if random_n_step:
                 actual_n_step = int(self.rng.integers(n_step[0], n_step[1]))
 
+            lookback = int(episode_ts != 0)
```

Comment:
Based on our slack discussion, just some thoughts:
* Maybe the user should provide the lookback value?
* Even if the user requires a lookback of 10000, it shouldn't matter in filtering out episodes here, b/c if the episode is not reaching back that many timesteps, then that's also ok and the lookback will result in fill values, not actual episode values.
* Thus, in the if-block below, we should simply do:
```
if episode_ts + batch_length_T + (actual_n_step - 1) > len(episode):
    continue
```

Reply:
If no lookback is provided I guess we run into this line and error out: `https://github.com/ray-project/ray/blob/5669b479e13fe65308071f4de8b5b0763afa6aa7/rllib/connectors/common/add_states_from_episodes_to_batch.py#L374` 

We could set `fill=True`. The things is that the starting point is `0` but the length is `1`.